music21/humdrum/spineParser.py

```diff
@@ -2201,49 +2282,62 @@ def hdStringToNote(contents):
 
     # http://www.lib.virginia.edu/artsandmedia/dmmc/Music/Humdrum/kern_hlp.html#kern
 
-    # 3.2.1 Pitches and 3.3 Rests
-
-    matchedNote = re.search('([a-gA-G]+)', contents)
-    thisObject = None
+    # Determine duration part first to avoid making an unused duration
+    thisDuration = _hdStringToDuration(contents, defaultDuration)
```

Comment:
Is this line supposed to be something like:

```
if defaultDuration:
    thisDuration = defaultDuration
else:
    thisDuration = _hdStringToDuration(contents)
```

??? this is the part I'm most confused by.

Reply:
The idea is that the `defaultDuration` argument you pass will be `None` most of the time, but in the case of chords, you pass the duration object of the first note in the chord. Then, you check the second note's duration part of its string. If the second note has a duration part to its string, then you use that no matter what it is. It's only if the second note doesn't specify its own duration that we want to use the first note's duration. But using your if/else above, that would use the first note's duration (the `defaultDuration` argument) for all other notes in the chord no matter what. Maybe the problem is just a naming one, does `fallbackDuration` make more sense?

Reply:
Hi @mscuthbert does my explanation above clarify this? Would you be ok with calling it `fallbackDuration`? Passing the first note's duration to all other notes in that chord is key to fixing the bug explained in my "minimal example" given in a comment above. I'm on board with all your other comments, so if we can agree on this then I can update the pr.